substrate/frame/contracts/uapi/Cargo.toml

```diff
@@ -23,6 +23,9 @@ scale = { package = "parity-scale-codec", version = "3.6.1", default-features =
 [target.'cfg(target_arch = "riscv32")'.dependencies]
 polkavm-derive = '0.5.0'
 
+[package.metadata.docs.rs]
+targets = ["wasm32-unknown-unknown", "x86_64-unknown-linux-gnu"]
+
```

Comment:
Seems unrelated to this PR. Merge failure somehow?

Reply:
arg yes will remove thanks

Reply:
fyi updated slightly the xcm dispatchable so that it returns DispatchErrorWithPostInfo instead of DispatchError so that we don't run out of gas if ever we want to recover from these errors